src/components/AddingTodoItem/AddingTodoItem.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+type Props = {
+  title: string
+};
+
+export const AddingTodoItem: React.FC<Props> = React.memo(({ title }) => {
```

Comment:
But this component is almost same as TodoItem, why do you need separate constant for it? We can just reuse TodoItem component and pass needed data